docs/topics/clusterdefinitions.md

```diff
@@ -436,7 +436,6 @@ Below is a list of apiserver options that are _not_ currently user-configurable,
 | "--etcd-keyfile"                            | "/etc/kubernetes/certs/etcdclient.key"                                                  |
 | "--etcd-servers"                            | _calculated value that represents etcd servers_                                         |
 | "--profiling"                               | "false"                                                                                 |
-| "--repair-malformed-updates"                | "false"                                                                                 |
```

Comment:
I understand that this is removed for k8s 1.14. Does the removal of this have consequences for other k8s clusters?